variables.tf

```diff
@@ -590,3 +590,42 @@ variable "csi_driver_version" {
   description = "Set to csi-driver version."
   default     = "v0.1.0"
 }
+
+################################################################
+# Image upload variables (used only for uploading RHCOS image 
+# from cloud object storage to PowerVS catalog)
+################################################################
+variable "bucket_region_map" {
+  type = map(string)
+
+  default = {
+    "syd" : "au-syd",
+    "osa" : "jp-osa",
+    "tok" : "jp-tok",
+    "eu-de" : "eu-de",
+    "lon" : "eu-gb",
+    "tor" : "ca-tor",
+    "dal" : "us-south",
+    "sao" : "br-sao",
+    "us-east" : "us-east"
+  }
+}
```

Comment:
This map is not going to change and should be set as a local variable. We can add new regions in the future.